dashboard/app/models/user.rb

```diff
@@ -1511,7 +1512,8 @@ def can_view_student_ai_chat_messages?
 
   # Students
   def has_ai_tutor_access?
```

Comment:
maybe change this to `can_use_ai_tutor` and then outline logic of enabled/access/available more clearly somewhere

Reply:
Did a minor refactor here.